pkgs/by-name/de/decent-sampler/package.nix

```diff
@@ -1,32 +1,36 @@
-{ lib
-, stdenv
-, fetchzip
-, fetchurl
-, makeDesktopItem
-, copyDesktopItems
-, buildFHSEnv
-, alsa-lib
-, freetype
-, nghttp2
-, libX11
-, }:
+{
+  lib,
+  stdenv,
+  fetchzip,
+  fetchurl,
+  makeDesktopItem,
+  copyDesktopItems,
+  buildFHSEnv,
+  alsa-lib,
+  freetype,
+  nghttp2,
+  libX11,
+}:
 
 let
   pname = "decent-sampler";
-  version = "1.10.0";
+  version = "1.11.19";
 
   icon = fetchurl {
-    url = "https://archive.org/download/ds-256/DS256.png";
-    hash = "sha256-SV8zY5QJ6uRSrLuGTmT1zwGoIIXCV9GD2ZNiqK+i1Bc=";
+    url = "https://www.decentsamples.com/favicon.ico";
+    hash = "sha256-+dg+jhlIpfqWMD+R6Q4QASHOSvLjLcLA1TMYsC/Ixz0=";
   };
 
   decent-sampler = stdenv.mkDerivation {
     inherit pname version;
 
     src = fetchzip {
       # dropbox links: https://www.dropbox.com/sh/dwyry6xpy5uut07/AABBJ84bjTTSQWzXGG5TOQpfa\
-      url = "https://archive.org/download/decent-sampler-linux-static-download-mirror/Decent_Sampler-${version}-Linux-Static-x86_64.tar.gz";
-      hash = "sha256-KYCf/F2/ziuXDHim4FPZQBARiSywvQDJBzKbHua+3SM=";
+      urls = [
+        "https://archive.org/download/decent-sampler-linux-static-download-mirror/Decent_Sampler-${version}-Linux-Static-x86_64.tar.gz"
```

Comment:
I think it'd be better to not rely on the internet archive at all and only use the upstream dropbox. The IA is not a CDN.

Reply:
> I think it'd be better to not rely on the internet archive at all and only use the upstream dropbox. The IA is not a CDN.

Ok many thanks. I'll drop the IA mirror entirely.